Backends/CLX-fb/mirror.lisp

```diff
@@ -84,18 +76,20 @@
                             (setf y min-y))))))))))
     (map-over-region-set-regions fn dirty-r)))
 
-(defmethod image-mirror-to-x ((sheet clx-fb-mirror))
+(defun image-mirror-to-x (mirror)
   (declare (optimize speed))
+  (unless (typep mirror 'clx-fb-mirror)
+    (return-from image-mirror-to-x))
```

Comment:
Can we instead ensure that this is only called with proper arguments?

Reply:
fixed